.rubocop.yml

```diff
@@ -42,7 +42,6 @@ AllCops:
     - 'bin/spring'
     - 'bin/update'
     - 'db/schema.rb'
-    - 'spec/dummy/**/*'
```

Comment:
No, this was intentional. It's not a Rails Engine, so it doesn't have a dummy app for specs. It was a leftover from an older PR.